src/openfermion/__init__.py

```diff
@@ -17,16 +17,142 @@
 
 www.openfermion.org
 """
-from openfermion.ops import *
-from openfermion.chem import *
-from openfermion.circuits import *
-from openfermion.functionals import *
-from openfermion.hamiltonians import *
-from openfermion.linalg import *
-from openfermion.measurements import *
-from openfermion.testing import *
-from openfermion.third_party import *
-from openfermion.transforms import *
-from openfermion.utils import *
+from openfermion.chem import (make_atom, make_atomic_lattice, make_atomic_ring,
+                              angstroms_to_bohr, bohr_to_angstroms,
+                              MolecularData, name_molecule, geometry_from_file,
+                              load_molecular_hamiltonian, periodic_table,
+                              periodic_hash_table, periodic_polarization,
+                              geometry_from_pubchem, make_reduced_hamiltonian)
```

Comment:
If you add a comma after the last item and reformat then the formatter should list one symbol per line which is more visually appealing (and - and this is totally minor - allows you to count exported symbols by looking at line numbers). See this example [`__init__.py`](https://github.com/quantumlib/Cirq/blob/master/cirq/__init__.py) in cirq for example of what this looks like.

(Applies throughout.)

Reply:
Thanks for the tip on the formatter!